rollbar-spring-webmvc/build.gradle

```diff
@@ -0,0 +1,19 @@
+ext {
+    springWebmvcVersion = '5.2.6.RELEASE'
+    javaxServletVersion = '3.0.1'
+    junitVersion = '4.12'
+    mockitoVersion = '2.10.0'
+    springFrameworkBootTestVersion = '2.2.6.RELEASE'
+}
+
+dependencies {
+    compile project(':rollbar-java')
```

Comment:
We don't need this dependency and neither the https://github.com/rollbar/rollbar-java/pull/215/files#diff-c719e376167cbf3b5424e5397aae4ff7R14 

Also, I think that the `testCompile` are adding at a higher level as we declare the same for all integrations, at leas the juni and mockito ones.
The Spring boot related one is not needed as well as this doesn't depend on anything related with Spring boot.

Reply:
i think we need the javax.servlet import in the gradle file for the HTTP Servlet usage in resolveException